The identifier x, used at <do_err/unbound_id_alone, 1:0-1:1>, is not in scope